lib/location/scan_wifi.c

```diff
@@ -59,6 +59,42 @@ void scan_wifi_details_get(struct location_data_details *details)
 }
 #endif
 
+static int shutdown_wifi(struct net_if *iface)
+{
+	LOG_DBG("Shutting down Wi-Fi interface");
+	int ret;
+
+	if (!net_if_is_admin_up(iface)) {
+		return 0;
+	}
+
+	ret = net_if_down(iface);
+	if (ret) {
+		LOG_ERR("Cannot bring down wifi iface (%d)", ret);
+		return ret;
+	}
+
+	LOG_DBG("Interface down");
+
+	return 0;
+}
+
+static int startup_wifi(struct net_if *iface)
```

Comment:
Maybe we could be consistent with naming and prefix functions with `scan_wifi`?
Also for shutdown.